docs/product/alerts/notifications/index.mdx

```diff
@@ -49,14 +49,35 @@ Sentry sends deploy notifications to users who have committed to the release tha
 
 ## Spend Notifications
 
-By default, Sentry sends spend notifications to all owners of an organization when:
+By default, Sentry sends spend notifications to all owners and billing members of an organization when:
 
-- 80% of the organization's reserved volume (for errors, transactions, and/or attachments) has been depleted.
-- The entire reserved volume and any on-demand budget for a category (such as errors, transactions, and/or attachments) has been depleted.
-- 80% of the organization's on-demand budget has been used.
+- 80% of the organization's reserved volume (such as errors, spans, and/or attachments) has been depleted.
+- The entire reserved volume and any pay-as-you-go budget for a category (such as errors, spans, and/or attachments) has been depleted.
+- 80% of the organization's pay-as-you-go budget has been used.
 
 Learn more in the [full Quotas documentation](/pricing/quotas/).
 
+### Configuring Spend Notifications
+
+Billing or owner members can set up their notification thresholds by going to: **Settings > Subscriptions > Notifications**.
+
+![Notifications tab of Subscription settings page](./img/subscription_notifications_settings.png)
+
+From here, you can set up customized notification thresholds for your organization:
+
+- **Subscription Consumption**: Set the percentage of the quota usage threshold(s) that triggers a notification.
+- **Pay-as-you-go Consumption**: Set the percentage of Pay-as-you-go usage threshold(s) that triggers a notification.
+
+### Personal Spend Notifications
+
+If you are an owner or a billing member of an organization, and if you wish to customize how you receive spend notifications, you can do so in your user's notification settings page:
+
+1. Go to your user setting's notifications page: https://sentry.sentry.io/settings/account/notifications/
+2. Click on the cog wheel icon on the **"Spend"** row.
+3. This should take you to: https://sentry.io/settings/account/notifications/quota/
+4. From here you can either enable or disable receiving spend notifications by delivery method (e.g. email or Slack), or by organization.
+    - **NOTE: **The configuration of the spend notifications thresholds (e.g. receive notifications on at least 80% quota consumed) is done on the Subscription settings page of the organization. See above on how to configure this.
+
```

Comment:
```suggestion
Owner and billing members of an organization can also customize how they personally receive spend notifications. This can be done from the user's notification settings page:

1. Go to the [**Notifications**](https://sentry.io/orgredirect/organizations/:orgslug/settings/account/notifications/) page (in your **User Settings**). 
2. Scroll to the **Spend** row and click on the cog wheel. 
3. From the [**Spend Notifications**](https://sentry.io/orgredirect/organizations/:orgslug/settings/account/notifications/quota/) page, you'll be able to select the types of notifications you get as well as how you'd like to get them (via Slack, email, and so on).

```
Wow, this is a pretty convoluted workflow :p - might be helpful to add a screenshot of the Spend Notifications page.